test/e2e/common/application_autoscaling_utils.py

```diff
@@ -0,0 +1,71 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: Change this file name to `utils.py`. I saw the conversation on the previous PR about this, but I agree with Suraj that the default for this repository is application autoscaling, so no need to preface anything with that.

Reply:
done